### PR TITLE
Ubuntu/Debian + RPM packaging for usrsctp-neat

### PR DESCRIPTION
**Pull request by dreibh**
This pull request add the files and scripts for Ubuntu/Debian + RPM packaging for usrsctp-neat. It is a merge of my upstream pull requests for usrsctp-sctplab (https://github.com/sctplab/usrsctp/pull/117 and https://github.com/sctplab/usrsctp/pull/116).

### CODE REVIEW

**Review by ruengeler — CHANGES_REQUESTED** [review]
@dreibh 
Hi Thomas,
the master branch should be synchronous to the master branch of usrsctp. I am waiting for Michael to merge some pull requests in usrsctp before merging the changes of the branch usrsctpForNeat into the master. 
Please compare your changes to usrsctpForNeat and create a new pull request.

**Comment by dreibh** [comment]
Updated pull request to work with latest version of the upstream-neat/master branch.

**Pull request merged**
